.github/actions/install-rust/action.yml

```diff
@@ -8,5 +8,20 @@ inputs:
     default: 'stable'
 
 runs:
-  using: 'node16'
-  main: 'main.js'
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        rustup show
+
+        # Save disk space by avoiding incremental compilation.
+        echo 'CARGO_INCREMENTAL=0' >> "$GITHUB_ENV"
+
+        # Turn down debuginfo from 2 to 1 to help save disk space
+        echo 'CARGO_PROFILE_DEV_DEBUG=1' >> "$GITHUB_ENV"
+        echo 'CARGO_PROFILE_TEST_DEBUG=1' >> "$GITHUB_ENV"
+
+    - if: ${{ ! startsWith(inputs.toolchain, 'nightly') }}
```

Comment:
Oh yeah, this reminds me: I broke the "toolchain" input when I switched this over to just reading `rust-toolchain.toml`. That input no longer controls the toolchain that's getting installed. So this test doesn't do what any reasonable person would think it's doing. I definitely agree this commit accurately translates `main.js` as it stands now into a composite action though!

Reply:
We should figure out if we'd like to keep this. It seems nice to have a reusable place to setup the environment variables we're relying on, but the action name seems a bit misleading now :)

Reply:
`rustup show` _does_ install the toolchain though, as a side-effect. :laughing: Maybe I should have commented to that effect when I made that change... It just has the nice bonus of recording in the CI log exactly what toolchain it ran with.